app/services/dialog_local_service.rb

```diff
@@ -126,6 +127,8 @@ def determine_api_endpoints(obj, display_options = {})
 
   def determine_target_type(obj)
     case obj.class.name.demodulize
+    when /ServiceContainerTemplate/
```

Comment:
I would suggest moving this down to be with `when /ServiceAnsiblePlaybook/` so that it's just: `when /ServiceAnsiblePlaybook/, /ServiceContainerTemplate/` and then for `/Template/`, do like I'm doing here in [this PR](https://github.com/ManageIQ/manageiq-ui-classic/pull/4374/files#diff-98745ecfd778f683f411147c83272a40R130):

`when /^Template/`.